### DISCUSSION TITLE
error converting YAML to JSON

### DISCUSSION THREAD

**Discussion by felipejareta** [fluxcd/flux2#3123] [General]
Hi everybody.

I have a fluxcd in my Openshift cluster. I don't know how and when but, every new app that I push to github, on flux folders, nothing happens. Even if I run flux reconcile ks <appfolder>, don't work.
So I check running flux get ks and receive this:

_NAME            REVISION        SUSPENDED       READY   MESSAGE
elasticsearch   main/373c0c7    False           True    Applied revision: main/373c0c7
flux-system                     False           False   error converting YAML to JSON: yaml: line 49: did not find expected key
keycloak        main/373c0c7    False           True    Applied revision: main/373c0c7
kong            main/373c0c7    False           True    Applied revision: main/373c0c7
konga           main/373c0c7    False           True    Applied revision: main/373c0c7
nexus           main/373c0c7    False           True    Applied revision: main/373c0c7
postgresql      main/373c0c7    False           True    Applied revision: main/373c0c7
rabbitmq        main/373c0c7    False           True    Applied revision: main/373c0c7
redis           main/373c0c7    False           True    Applied revision: main/373c0c7
service-mesh    main/373c0c7    False           True    Applied revision: main/373c0c7
sonarqube       main/373c0c7    False           True    Applied revision: main/373c0c7_

I checked all flux-system files, kustomizations files, as well I compared with other environment that flux work, but I didn't find anything.
Anyway, if I run flux reconcile or any other command like resume or suspend , apparently it run without error, but don't build anything.

Is someone knows what can be this?

Regards.

**Comment by stefanprodan** [comment]
Can you please run this script to validate your manifests https://github.com/fluxcd/flux2-kustomize-helm-example/blob/main/scripts/validate.sh

**Reply by felipejareta** [reply]
Hi @stefanprodan I don't have this prerequisits in my environment. My openshift cluster is thin and some tools are blocked.
I forgot to mentionate my flux version is 0.30.2
When I try to run script, I receive an error about yq (command not found).
Is there any other thing that I can do?
